indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -105,6 +105,20 @@
       overflow-wrap: anywhere;
     }
 
+    :global(.field.error) .boolean-field :global(.ui.buttons) {
+      :global(.ui.button) {
+        background-color: #fff6f6;
+        color: $dark-red;
+        outline: 1px solid $red;
```

Comment:
Can we use the same color as SUI? `#CC0000`

Also, maybe we can move this rule down to the other boolean-input related rule?:
https://github.com/indico/indico/blob/398aeb2706435088d64f174977069541af4b71ce/indico/modules/events/registration/client/styles/regform.module.scss#L130-L133